features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -49,13 +49,12 @@ Feature: Gift Exchange Challenge
     Then I should see "My Gift Exchange"
 
   Scenario: Change timezone for a gift exchange
-    Given I am logged in as "mod1"
-      And I have created the gift exchange "My Gift Exchange"
-      And I am on "My Gift Exchange" gift exchange edit page
-    When I select "(GMT-08:00) Pacific Time (US & Canada)" from "gift_exchange_time_zone"
-    And I submit
-      Then I should see "Challenge was successfully updated"
-      Then I should see the correct time zone for "Pacific Time (US & Canada)"
+    Given the gift exchange "My Gift Exchange" is ready for signups
+    When I go to "My Gift Exchange" gift exchange edit page
+      And I select "(GMT-08:00) Pacific Time (US & Canada)" from "Time zone"
+      And I submit
+    Then I should see "Challenge was successfully updated"
+      And I should see "PDT"
```

Comment:
This should remain `Then I should see the correct time zone for "Pacific Time (US & Canada)"`.